File: lab02_stack/task01/scripts/subscriber_node.py
#!/usr/bin/env python
import rospy
##TODO: include the library for a msg for an integer

##TODO: complete the callback function. Your callback function should print out the incoming message.

def callback(data):
    print 'Delete this line.'

def listener():

    rospy.init_node('listener', anonymous=True)

    ##TODO: Define a nodehandle and a subscriber.

    rospy.spin()


if __name__ == '__main__':
    listener()

File: lab02/lab02task01/src/subscriber_node.cpp
#include "ros/ros.h"
//TODO: include the library for a msg for an integer

//TODO: complete the callback function. Your callback function should print out the incoming message.
void subscribeCallback();

int main(int argc, char **argv)
{
    ros::init(argc, argv, "listener");

    //TODO: Define a nodehandle and a subscriber.
    ros::spin();

    return 0;
}